features/keystone/identity/ldap.pan

```diff
@@ -9,18 +9,14 @@ include 'components/metaconfig/config';
 prefix '/software/components/metaconfig/services/{/etc/keystone/keystone.conf}';
 'contents/identity/domain_specific_drivers_enabled' = 'True';
 'contents/identity/domain_config_dir' = '/etc/keystone/domains';
-'contents/identity/driver' = 'ldap';
+'contents/identity/driver' = 'sql';
 'contents/resource/driver' = 'sql';
 'contents/assignment/driver' = 'sql';
 
 # All ldap configuration is put on /etc/keystone/domains/keystone.DOMAIN_NAME.conf
 prefix '/software/components/metaconfig';
 'services' = {
-  # default domain is still on SQL
```

Comment:
the problem when you remove this block is that you should put 
variable OPENSTACK_KEYSTONE_IDENTITY_LDAP_PARAMS ?= dict('default', 'blahblah');
on openstack/default/config.pan

and be sure that user will not remove it.